hawc_hal/HAL.py

```diff
@@ -410,9 +410,9 @@ def display_spectrum(self):
         return self._plot_spectrum(net_counts, yerr, model_only, residuals, residuals_err)
 
     def _plot_spectrum(self, net_counts, yerr, model_only, residuals, residuals_err):
-
+        plt.rcParams["figure.figsize"] = [12, 6]
         fig, subs = plt.subplots(2, 1, gridspec_kw={'height_ratios': [2, 1], 'hspace': 0})
```

Comment:
Changing `plt.rcParams` will affect the defaults for all future plots in the same python session, so generally that's not something we'd want to change in a function like this. However, you can give the figure size as an argument to the `subplots` command, e.g. `fig, subs = plt.subplots( ..., figsize=(12,6) )`. That way, only the one figure will be affected.

Reply:
Sure, I'll make that change now (it returns the same plot format).  In the meantime, these are the 2 outcomes depending on whether fhit or gp is used.

![hal_crab_residuals_gp](https://user-images.githubusercontent.com/73908793/113904063-43e6ff00-9797-11eb-8cae-1b7d2735130b.png)
![hal_crab_residuals](https://user-images.githubusercontent.com/73908793/113904067-43e6ff00-9797-11eb-9b7b-8e442a4b304f.png)